### PR TITLE
Allow larger  recurrent states

### PR DESCRIPTION
**Pull request by Ruishenl**

### CODE REVIEW

**Comment by Ruishenl** [comment]
Did a pre release 0.0.15dev0 to test first

File: invertedai_cpp/invertedai/session.h [hunk 1]
```diff
@@ -27,7 +28,7 @@ class Session {
 public:
   const char* host_ = local_mode ? "localhost" : "api.inverted.ai";
   const char* port_ = local_mode ? "8000" : "443";
-  const char *subdomain = local_mode ? "/" : "/v0/aws/m1/";;
+  const char *subdomain = local_mode ? "/" : "/staging/aws/m1/";;
```
Comment (adscib) [inline invertedai_cpp/invertedai/session.h:31]: Is this meant to go in?

**Review by adscib — COMMENTED** [review]
Just one thing to double check, otherwise looks good.

File: invertedai_cpp/invertedai/session.h (hunk 1 above)
Comment (Ruishenl) [reply invertedai_cpp/invertedai/session.h]: Thanks, no definelty not

**Review by adscib — APPROVED** [review]

**Pull request merged**